itou/www/apply/forms.py

```diff
@@ -360,6 +360,12 @@ class SubmitJobApplicationForm(forms.ModelForm, ResumeFormMixin):
     Submit a job application to an SIAE.
     """
 
+    class Meta:
+        model = JobApplication
+        fields = ["selected_jobs", "message"] + ResumeFormMixin.Meta.fields
+        widgets = {"selected_jobs": forms.CheckboxSelectMultiple()}
+        labels = {"selected_jobs": "Métiers recherchés"}
```

Comment:
Quitte à réordonner, je dirais que si le `model` peut mériter d'être en premier malgré :abcd: (et encore :suspect: ) je dirais que les attributes suivants pourraient gentiment suivre l'ordre alphabétique.

```suggestion
        labels = {"selected_jobs": "Métiers recherchés"}
        widgets = {"selected_jobs": forms.CheckboxSelectMultiple()}
```

Reply:
Je ne suis pas sûr qu’on arrivera à le maintenir dans la durée, et il y a une hierarchisation des infos par importance. Genre `model` avant `fields`. Mais je refais un petit tour pour voir ceux qu’on peut mieux ordonner. :+1:

Reply:
Et voilà !

Reply:
Je ne vois pas la modif mais je suis d'accord avec toi que cela s'annonce compliqué à maintenir dans la durée (tout comme l'ordre au niveau du ModelForm :stuck_out_tongue_winking_eye:   )

Reply:
L’ordre au niveau de la `class Meta:` m’importe plus, car les infos sont pénibles à chercher autrement, surtout avec des méthodes `clean` velues.